Bitbucket.Authentication/Src/Rest/RestClient.cs

```diff
@@ -26,7 +26,9 @@ public static string UserUrl
                 Authorization = authorization,
                 Timeout = TimeSpan.FromMilliseconds(requestTimeout),
             };
-            var requestUri = targetUri.CreateWith(restRootUrl);
+
+            var apiUrl = new Uri(restRootUrl, UserUrl);
+            var requestUri = new TargetUri(apiUrl, targetUri.ProxyUri);
```

Comment:
Make sure we are calling the https://api.bitbucket.org/2.0/user REST resource

Reply:
I fixed this up to use the `TargetUri.CreateWith` method.